### ISSUE TITLE
build error on ts monorepo

### ISSUE THREAD

**Issue by himself65** [huozhi/bunchee#386]
https://github.com/himself65/bunchee-issue/tree/issue-1

```json
{
  "name": "core",
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "require": "./dist/index.js",
      "import": "./dist/index.js"
    }
  },
  "scripts": {
    "build": "bunchee"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "bunchee": "^4.3.3"
  }
}

```

```shell
cd packages/core
pnpm run build
```

```
/Users/himself65/Library/Caches/fnm_multishells/9583_1705091499933/bin/npm run build

> build
> bunchee

error TS6379: Composite projects may not disable incremental compilation.

⨯ RollupError: Failed to compile. Check the logs above.
    at Object.error (/Users/himself65/Code/bunchee-monorepo/node_modules/.pnpm/rollup@4.9.5/node_modules/rollup/dist/shared/parseAst.js:279:30)
    at Object.error (/Users/himself65/Code/bunchee-monorepo/node_modules/.pnpm/rollup@4.9.5/node_modules/rollup/dist/shared/rollup.js:820:32)
    at Object.error (/Users/himself65/Code/bunchee-monorepo/node_modules/.pnpm/rollup@4.9.5/node_modules/rollup/dist/shared/rollup.js:18666:42)
    at generateDtsFromTs (/Users/himself65/Code/bunchee-monorepo/node_modules/.pnpm/rollup-plugin-dts@6.1.0_rollup@4.9.5_typescript@5.3.3/node_modules/rollup-plugin-dts/dist/rollup-plugin-dts.cjs:1857:30)
    at Object.transform (/Users/himself65/Code/bunchee-monorepo/node_modules/.pnpm/rollup-plugin-dts@6.1.0_rollup@4.9.5_typescript@5.3.3/node_modules/rollup-plugin-dts/dist/rollup-plugin-dts.cjs:1866:38)
    at /Users/himself65/Code/bunchee-monorepo/node_modules/.pnpm/rollup@4.9.5/node_modules/rollup/dist/shared/rollup.js:1005:40

Process finished with exit code 1

```

**Comment by huozhi** [comment]
Looks like i can't disable the `incremental` config for dts generation as it still can work and disabling might breaks ts rules

**Comment by huozhi** [comment]
Fixed in 4.3.4
